components/sysutils/exfat/patches/libexfat-platform.h.patch

```diff
@@ -0,0 +1,19 @@
+--- tries/libexfat/platform.h.orig	2021-12-13 10:39:39.520746794 +0000
```

Comment:
Can you make a comment here to describe the reason why this patch exists so we can make decisions if we want to keep it or not in the future?

Reply:
Add macroes to describe the endianness of the platform used, and to define how bytes are to be swapped accordingly.